### ISSUE TITLE
Plane: wp_negative is giving radius 30 meter regardless the input number.

### ISSUE THREAD

**Issue by Elios007** [ArduPilot/ardupilot#3919]
#### Issue details
we were doing loiter today but we found that when wp_loiter_rad is positive radius is correct but when negative for a counter clockwise loiter, the radius is very small (30m) regardless the input number. We tried even -5000.

#### Version
3.5.2

#### Platform
- [ ] Plane

#### Airframe type
Fixed wings plane

#### Hardware type
Pixhawk

#### Logs
https://www.dropbox.com/sh/hkt7zy6iuu8dtb2/AADARqvwErd4tKfHBBRAO74Ka?dl=0

**Comment by magicrub** [comment]
I believe this was fixed by Michael dB just after the v3.5.2 release, will
check and follow up.
On Apr 17, 2016 2:47 PM, "Elios" <notifications@github.com> wrote:

Issue details

we were doing loiter today but we found that when wp_loiter_rad is positive
radius is correct but when negative for a counter clockwise loiter, the
radius is very small (30m) regardless the input number. We tried even -5000.
Version

3.5.2
Platform

   - Plane

Airframe type

Fixed wings plane
Hardware type

Pixhawk
Logs

https://www.dropbox.com/sh/hkt7zy6iuu8dtb2/AADARqvwErd4tKfHBBRAO74Ka?dl=0

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/ArduPilot/ardupilot/issues/3919>
